### PR TITLE
Moving Heidi's and Camila's personas to contributors file

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #1190 and #1106 

* Content from these two PRs are added to the contributors.md file to be updated online

### List of changes proposed in this PR (pull-request)

* [ ] Move Heidi's persona to the contributors file
* [ ] Move Camila's persona to the contributors file

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@CamillaBressan Can I ping you for this PR to check you are happy with your profile to be moved to the contributors.md?

**Comment by crangelsmith** [comment]
> @CamillaBressan and @HeidiSeibold Can I ping you for this PR to check you are happy with your profile to be moved to the contributors.md? @HeidiSeibold You might need to update your position too :D

Hi @malvikasharan, I suspect to meant to tag me on this (instead of Camilla Bressan?). If so i'm happy with getting my profile moved.

**Comment by malvikasharan** [comment]
Thanks Camila. Yes I mean to tag you 💯 Can you please approve this PR so I can merge your profile?

**Review by crangelsmith — APPROVED** [review]

**Comment by malvikasharan** [comment]
@sgibson91 I am not sure Latin bot is reading a file that it is supposed to skip :(

The error of etc is in the contributors-record.md which I have added in the files list to skip. Maybe you can help me troubleshoot this whenever you have some time:
`
IGNORE_LIST = [
    os.path.join(ABSOLUTE_HERE, "book/website/_config.yml"),
    os.path.join(ABSOLUTE_HERE, "book/website/community-handbook/style.md"),
    os.path.join(ABSOLUTE_HERE, "book/website/afterword/contributors-record.md")
]
`
image

**Comment by sgibson91** [comment]
I would try:

```
IGNORE_LIST = [
    os.path.join(ABSOLUTE_HERE, "book", "website", "_config.yml"),
    etc...
]
```

In the screenshot, there also seems to be two `/the-turing-way/`'s in the filepath, so I wonder if `ABSOLUTE_HERE` behaves differently locally to on CI?

File: book/website/afterword/contributors-record.md [hunk 1]
```diff
@@ -109,6 +102,76 @@ I am a Mozilla fellowship (2016) and Fulbright scholarship (2007) alumna.
...
+> I’m the co-founder of the CEVALE2VE project (http://www.cevale2ve.org/en/home/), which of a virtual learning community that aims to tackle the serious issue of brain-drain in some Latin-American countries by bringing back the knowledge in a digital/online platform.
+> More recently that project has become consolidated into LA-CoNGA physics (http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
+> In this project I’m helping to build a data science module that will be thought in an online master course.
+
+### Heidi Seibold
+
+* GitHub id: [HeidiSeibold](https://github.com/HeidiSeibold)
+* ORCID id: [0000-0002-8960-9642](https://orcid.org/0000-0002-8960-9642)
+* Short bio:
+> I am a medical AI researcher, research software engineer and open science expert at LMU Munich, Bielefeld University and the Helmholtz Zentrum Munich.
```
Comment (HeidiSeibold) [inline book/website/afterword/contributors-record.md:135]:
```suggestion
> I lead a group on Open AI in Health at the Helmholtz Zentrum Munich.
```

**Review by HeidiSeibold — COMMENTED** [review]
Thanks @malvikasharan. I updated my affiliation. Otherwise looks good!

**Pull request merged**